src/category_theory/discrete_category.lean

```diff
@@ -39,7 +39,21 @@ universes v₁ v₂ v₃ u₁ u₂ u₃
 A type synonym for promoting any type to a category,
 with the only morphisms being equalities.
```

Comment:
Is "type synonym" the correct description now? It sort of makes sense, but just in case wrapping a type in a structure has other terminology, I thought I'd flag it.

Reply:
Thanks. I've cleaned up the comments here and above, now.